File: fsAndroid/MainActivity1.fs
namespace App1

open System
open Android.App
open Android.Media

type Resources = App1.Resource

[<Activity (Label = "MainActivity1", MainLauncher = true)>]
type MainActivity1 () =
  inherit Activity()

  override this.OnCreate(bundle) =
    base.OnCreate (bundle)
    // Create your application here
    this.SetContentView (Resources.Layout.layout1)

    let mutable mediaPlayer : MediaPlayer = MediaPlayer.Create(this, Resources.Raw.christmas)
    mediaPlayer.Start()

File: csAndroid/arch/MainActivity1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.Media;

namespace App1
{
    [Activity(Label = "MainActivity1", MainLauncher = true)]
    public class MainActivity1 : Activity
    {
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);
            SetContentView(Resource.Layout.layout1);

            MediaPlayer mediaPlayer = MediaPlayer.Create(ApplicationContext, Resource.Raw.christmas);
            mediaPlayer.Start();
        }
    }
}
